js/jquery.mapael.js

```diff
@@ -673,12 +673,12 @@
                 $map.unbind("resizeEnd", createPlotLegend);
```

Comment:
Shouldn't be replaced by `$map.unbind("resizeEnd." + pluginName, createPlotLegend);` instead ?